icu4j/readme.html

```diff
@@ -14,24 +14,24 @@
 <body style="background-color: rgb(255, 255, 255);" lang="EN-US"
  link="#0000ff" vlink="#800080">
 <h1>International Components for Unicode for Java (ICU4J)</h1>
-<h2>Read Me for ICU4J 67.1</h2>
-(Last Update: 2020-Apr-1)
+<h2>Read Me for ICU4J 68.1</h2>
+(Last Update: 2020-Oct-1)
 <hr size="2" width="100%">
 
 <p>
-<b>Note:</b> This is major release of ICU4J. It contains bug fixes and adds implementations
-of inherited API and introduces new API or functionality.
-<!-- <b>Note:</b> This is a preview release of ICU4J 67.
+<!-- <b>Note:</b> This is major release of ICU4J. It contains bug fixes and adds implementations
+of inherited API and introduces new API or functionality. -->
```

Comment:
If we make the icu4c readme the "final" version, then we should do the same for the icu4j readme here to be consistent.

Reply:
Done.  Thanks for catching that.